deployments/prod/environment.py

```diff
@@ -39,9 +39,11 @@ def env() -> Mapping[str, Optional[str]]:
         'AZUL_S3_BUCKET': 'edu-ucsc-gi-azul-dcp2-prod-storage-{AZUL_DEPLOYMENT_STAGE}',
 
         'AZUL_CATALOGS': ','.join([
-            f'hca:{name}{rel}:repository/tdr:metadata/hca'
-            for rel in (6, 1)
-            for name in ('dcp', 'it')
+            f"{project}:{'it' + it_name if it else name}:repository/tdr:metadata/hca"
```

Comment:
`itlungmap` won't be detected as an IT catalog name.

"project" is called "atlas"

Your version makes it harder to add a catalog since the 6 and the 1 are now duplicated.

Reply:
Try this instead:

```diff
Index: deployments/prod/environment.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/deployments/prod/environment.py b/deployments/prod/environment.py
--- a/deployments/prod/environment.py	(revision 445cd50de9a6b9b3893b0f77583ab7fc7eeae30a)
+++ b/deployments/prod/environment.py	(date 1624319570964)
@@ -42,6 +42,9 @@
             f'hca:{name}{rel}:repository/tdr:metadata/hca'
             for rel in (6, 1)
             for name in ('dcp', 'it')
+        ] + [
+            f'lungmap:{name}:repository/tdr:metadata/hca'
+            for name in ('lungmap', 'it0lungmap')
         ]),
 
         'AZUL_TDR_SOURCES': ','.join([
```

Reply:
Part of the reason I didn't implement like this initially was because I thought it may be important to define all of the non-`it` catalogs before the `it` catalogs. I got this impressions because they are ordered this way in all of the `environment.py`s I checked.

Is this something you considered?

Reply:
No. The constraint I mentioned is the relevant one.

> Your version makes it harder to add a catalog since the 6 and the 1 are now duplicated.